aws-java-sdk-s3/src/main/java/com/amazonaws/services/s3/transfer/DownloadCallable.java

```diff
@@ -217,7 +224,7 @@ private void combineFiles() throws Exception {
      * operation), adjust the file length so that the part starts writing from
      * the correct position.
      */
-    private void truncateDestinationFileIfNecessary() throws IOException {
+    private void truncateDestinationFileIfNecessary() {
```

Comment:
IOException was not thrown from this method. It only thrown unchecked exception.